[Dashboard] Update the Ray dashboard documentation to explain memory view. (#8945)
ray-project/ray: doc/source/ray-dashboard.rst

.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/dashboard/Logical-view-basic.png
    :align: center

Memory View
~~~~~~~~~~~~

> The Memory view shows you:

Reviewer: ```suggestion
The memory view shows you:
```

> - States of Ray objects, such as sizes, refernece types, and call sites.

Reviewer: ```suggestion
- The state of Ray objects, including their size, reference type, and call site.
```

> - Summary of reference types and object sizes in use.

Reviewer: ```suggestion
- A summary of reference types and object sizes in use.
```

You can also see it is infeasible to create ``Actor2`` because it requires 4 GPUs which 
is bigger than the total gpus available in this cluster (2 GPUs). 

Debugging ObjectStoreFullError and Memory Leak
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> You can get information for Ray objects through the Ray Memory tab. It is useful to debug memory leak, especially the `ObjectStoreFullError` error.

Reviewer: ```suggestion
You can view information for Ray objects in the memory tab. It is useful to debug memory leaks, especially `ObjectStoreFullError`.
```

> Note that this is the ported version of the `ray memory command <https://docs.ray.io/en/latest/memory-management.html#debugging-using-ray-memory>`_. To learn how to translate this table, please learn the `ray memory` command.

Reviewer: ```suggestion
Note that this is the same information as displayed in the `ray memory command <https://docs.ray.io/en/latest/memory-management.html#debugging-using-ray-memory>`_. For details about the information contained in the table, please see the `ray memory` documentation.
```
